docs/Hosts/Spawn-Hosts.md

```diff
@@ -18,6 +18,18 @@ If you are having trouble connecting to a spawn host:
       PubkeyAcceptedAlgorithms +ssh-rsa
 ```
 
+## Hosting a server on a spawn host
+
+To connect to your spawn host over a port (to, for example, test a UI or API),
+please use SSH port forwarding:
+
+```sh
+ssh -L <local_port>:localhost:<remote_port> <user>@<host>
```

Comment:
```suggestion
ssh -L <local_port>:localhost:<remote_port> <user>@<spawnhost>
```
I sometimes get confused about which host goes where; `<remotehost>` would also be unambiguous.

Reply:
Changed to `remote_host` to mirror `remote_port` earlier. Thanks!